### PR TITLE
Adding Ophys Scheduling

### PR DESCRIPTION
**Pull request by Ahad-Allen**
This PR is intended to allow for ophys jobs to wait until previous jobs have been completed, preventing issues with filling hard drives while generating very large nwb files.

### CODE REVIEW

**Comment by Ahad-Allen** [comment]
Feel free to ignore the first comment, that is more related to quick fixes that were done to fix issues with updating the NWB files. 
This PR works by tracking the current job id, storing it, and then using that as an input for SLURM jobs. The jobs will not trigger until this ID has finished.

**Review by jeromelecoq — COMMENTED** [review]
This is clean. I understood you are pulling the job environment variable from SLURM (presumably the variable is set by the job scheduler?) and pushing it to firebase so that the next job can use it. 

Side question, how are firebase credentials shared with the cluster job?

File: scripts/cron.py [hunk 1]
```diff
@@ -143,12 +144,14 @@
         # FB info on dandi locations
         for experiment in exp_list:
             if experiment == exp_list[-1]:
-                cmd = dir + '/bash/raw_ophys.sh ' + " -p " + project + + "-s " + str(session) + " -e " + str(
+                cmd = dir + '/bash/raw_ophys.sh ' + " -p " + project   + " -s " + str(session) + " -e " + str(
```
Comment (rcpeene) [inline scripts/cron.py:147]: 🤢

Comment (rcpeene) [inline scripts/cron.py:147]:
```suggestion
                cmd = dir + '/bash/raw_ophys.sh ' + " -p " + project + " -s " + str(session) + " -e " + str(

```
🤢

**Comment by Ahad-Allen** [comment]
> This is clean. I understood you are pulling the job environment variable from SLURM (presumably the variable is set by the job scheduler?) and pushing it to firebase so that the next job can use it.
> 
> Side question, how are firebase credentials shared with the cluster job?

There is a cred file in the repo that's used for the credentials for firebase and lims. When the firebase init is called, it looks for that specific file to get write access :)

**Pull request merged**